kairon/nlu/classifiers/llm.py

```
@@ -282,3 +282,21 @@ def persist(self) -> None:
+        """Adds this extractor's name to a list of entities.
+
+        Args:
+            entities: the extracted entities.
+
+        Returns:
+            the modified entities.
+        """
+        entities_new = []
+        for entity in entities:
+            if isinstance(entity, dict):
+                entity[EXTRACTOR] = self.name
```

Ruff (0.8.2)

Undefined name `EXTRACTOR`

(F821)